packages/slice-machine/lib/builders/SliceBuilder/Header/SimulatorButton/style.module.css

```diff
@@ -0,0 +1,15 @@
+div.tooltip {
```

Comment:
i didn't know this was an option, does it support scss?

Reply:
I think it's supported OOTB by Next yes!